v3/src/components/graph/adornments/adornments.tsx

```diff
@@ -16,7 +16,7 @@ export const Adornments = observer(function Adornments() {
     layout = useGraphLayoutContext(),
     adornments = graphModel.adornments
 
-  if (adornments.length === 0) return null
+  if (adornments?.length === 0) return null
```

Comment:
```suggestion
  if (!adornments?.length) return null
```
Otherwise, the test would fail if `adornments` is `undefined`.

Reply:
Ah, right. Thanks 👍